ad_hoc_bench/Cargo.toml

```diff
@@ -18,7 +18,8 @@ fork_choice_control = { workspace = true }
 fork_choice_store = { workspace = true }
 futures = { workspace = true }
 itertools = { workspace = true }
-tracing ={ workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order